### ISSUE TITLE
Merge variables objects when flushing queue

### ISSUE THREAD

**Issue by jasimon** [helfer/apollo-link-debounce#10]
Would you be ok with merging the `variables` object for all queued requests when flushing the queue?  It seems to me that for cases where you're updating an object instead of a single value that would be the desired behaviour.  Happy to take a crack at it if you're open.

**Comment by helfer** [comment]
I think that could be useful, but it should be optional, and ideally configurable per debounce key, so one debounce link could debounce requests both with and without merging variables.

**Comment by jasimon** [comment]
Looks like I don't have permissions to push to the repo?  I took a pass at it, curious to get your thoughts.

**Comment by helfer** [comment]
Yeah, you'll have to push to your own fork and then open a pull request.

-------- Original Message --------

**Comment by jasimon** [comment]
Sorry for the delay, PR here: https://github.com/helfer/apollo-link-debounce/pull/11
